content/basic-concepts/dvc-project.md

```diff
@@ -0,0 +1,19 @@
+---
+name: 'DVC Project'
+match:
+  [
+    'DVC project',
+    'DVC projects',
+    project,
+    projects,
+    'DVC repository',
+    'DVC repositories',
+    repository,
+    repositories
+  ]
+tooltip: >-
+  Initialized by running `dvc init` in the **workspace** (typically a Git
+  repository). It will contain the `.dvc/` directory, as well as `dvc.yaml` and
+  `.dvc` files created with commands such as `dvc add` or `dvc stage add`.  
+  [More info](/doc/user-guide/basic-concepts/dvc-project)
```

Comment:
Now that these have their own files we should move this into the MD body and use the proper MD processing to handle it, but that'll take an edit to `gatsby-theme-iterative`.